link/kprobe_test.go

```diff
@@ -37,6 +37,10 @@ func TestKprobe(t *testing.T) {
 	c.Assert(err, qt.IsNil)
 	defer k.Close()
 
+	pe, ok := k.(*perfEvent)
```

Comment:
No need for these assertions, `eventType` should be checked in `TestKprobeCreatePMU` and `TestKprobeTraceFS` like it was before.